bokehjs/package-lock.json

```diff
@@ -660,22 +660,19 @@
       "version": "4.0.0",
       "resolved": "https://registry.npmjs.org/arr-diff/-/arr-diff-4.0.0.tgz",
       "integrity": "sha1-1kYQdP6/7HHn4VI1dhoyml3HxSA=",
-      "dev": true,
-      "optional": true
+      "dev": true
```

Comment:
Please revert all changes to this file and use e.g. `git add -p` in future to avoid committing unrelated changes.